core/domain/exp_domain.py

```diff
@@ -585,6 +605,11 @@ def from_dict(
                 state_domain.WrittenTranslations.from_dict(
```

Comment:
@aks681, I'm not sure why don't we use State.from_dict over here, is there any specific reason?